scripts/svm/queryFills.ts

```diff
@@ -58,18 +59,22 @@ async function queryFills(): Promise<void> {
         { Property: "repaymentChainId", Value: event.data.repaymentChainId.toString() },
         { Property: "originChainId", Value: event.data.originChainId.toString() },
         { Property: "depositId", Value: event.data.depositId.toString() },
+        { Property: "depositIdNum", Value: u8Array32ToInt(event.data.depositId).toString() },
         { Property: "fillDeadline", Value: event.data.fillDeadline.toString() },
         { Property: "exclusivityDeadline", Value: event.data.exclusivityDeadline.toString() },
         { Property: "exclusiveRelayer", Value: new PublicKey(event.data.exclusiveRelayer).toString() },
         { Property: "relayer", Value: new PublicKey(event.data.relayer).toString() },
         { Property: "depositor", Value: new PublicKey(event.data.depositor).toString() },
         { Property: "recipient", Value: new PublicKey(event.data.recipient).toString() },
-        { Property: "message", Value: event.data.message.toString() },
+        { Property: "messageHash", Value: new PublicKey(event.data.messageHash).toString() },
```

Comment:
why format as Pubkey? Its more consistent with emitted EVM event to log 32 byte long hex string